vanilla-framework/vanilla-framework
# 1443 + 1444: Fix media object icons on Firefox + Change setting of height and width

## Done

- Replaced instances of '#' with '%23' in the SVG urls in the media object icons
- Changed height and width rules so that aspect ratio is not ruined for non-square images

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/media-object/media-object/
- Check that the icons show in Firefox as well as Chrome
- Check that non-square images don't have aspect-ratio messed up (you can just change the url of the placeholder image)

## Details

Fixes #1443, #1444
